newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -775,6 +830,17 @@ export default class AuthenticatedUserProvider extends React.Component<
     this._automaticallyUpdateUserProfile = true;
   };
 
+  _doLoginOnDesktopApp = async () => {
```

Comment:
this function doesn't seem to do what its naming suggests.
Should it be `onLoginOnDesktopApp` ? or `doNotifyLoggedInOnDesktopApp`